### PR TITLE
Prepare for 1.5.3 release

### PR DESCRIPTION
**Pull request by cbgbt**
**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by zmrow — APPROVED** [review]
:field_hockey:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+# v1.5.3 (2022-01-25)
+
+## Security Fixes
+* Update Bottlerocket SDK to 0.25.1 for Rust 1.58.1 ([#1918])
+* Update kernel-5.4 and kernel-5.10 to include recent security fixes. ([#1921])
```
Comment (bcressey) [inline CHANGELOG.md]: nit: it seems like we don't typically include the final `.`
```suggestion
* Update kernel-5.4 and kernel-5.10 to include recent security fixes ([#1921])
```

**Review by bcressey — APPROVED** [review]

**Review by etungsten — APPROVED** [review]

**Comment by cbgbt** [comment]
^ Force pushed to fix the consistency issue pointed out by @bcressey.

**Review by jpculp — APPROVED** [review]

**Review by bcressey — APPROVED** [review]

**Review by webern — APPROVED** [review]

**Comment by cbgbt** [comment]
^ Updated changelog to include the fix in #1925.

**Review by zmrow — APPROVED** [review]
:lollipop:

**Review by bcressey — APPROVED** [review]

**Comment by cbgbt** [comment]
Seems like the github actions runners were having issues, hence the failing checks.

**Pull request merged**